sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -666,6 +673,231 @@ public void batchExecutionWithOptionsApi() {
         validateResponse(batchResponse.getResults().get(3).getItem(EncryptionPojo.class), createPojo);
     }
 
+    private int getTotalRequest() {
+        int countRequest = new Random().nextInt(100) + 120;
+        logger.info("Total count of request for this test case: " + countRequest);
+
+        return countRequest;
+    }
+
+    @Test(groups = {"encryption"}, timeOut = TIMEOUT)
+    public void bulkExecution_createItem() {
```

Comment:
We should be validating the response against original item, how we are sure that encryption decryption working fine ?
You have to use validateResponse used in other tests, if not on all the items at least on one.
This is applicable to all the new tests you added

Reply:
Added validate response for Create, Upsert and Read Item Test cases for bulk.

Reply:
Tested the code with 5-10k operations as well.